draft-ietf-scitt-architecture.md

```diff
@@ -267,15 +263,10 @@ The Statement is considered opaque to Transparency Service, and MAY be encrypted
 
 Subject:
 
-: This term has the same definition as in {{RFC8392}}, which relies on the definition in RFC7519.
-The `sub` (subject) claim identifies the principal that is the subject of the CWT.
-The claims in a CWT are normally statements about the subject.
-In SCITT, `sub` identifies the entity about which statements, and receipts are made.
-The subject value MUST either be scoped to be locally unique in the context of the Issuer or be globally unique.
-The processing of this claim is generally application specific.
-The `sub` value is a case-sensitive string containing a StringOrURI value.
-Issuers use `sub` to identify the entity about which they are making Signed Statements.
-Transparency Services use `sub` to identify the entity about which they are issuing a Receipt.
+: an identifier that represents the Artifact about which Statements (and Receipts) are made and by which a logical collection of Statements about the same Artifact can be grouped.
+It is possible that there are multiple Statements about the same Artifact.
+In these cases, Issuers use `Subject` to create a coherent sequence of Signed Statements about the same Artifact and Verifiers can leverage `Subject` to ensure completeness and Non-equivocation across Statements by identifying all Transparent Statements associated to a specific one.
```

Comment:
```suggestion
In these cases, distinct Issuers might agree to use `Subject` to create a coherent sequence of Signed Statements about the same Artifact and Verifiers can leverage `Subject` to ensure completeness and Non-equivocation across Statements by identifying all Transparent Statements associated to a specific one.
```

Reply:
Add an example after this, where the issuers are library authors and the subject is a library with multiple dependencies.